Pcap++/src/DpdkDevice.cpp

```diff
@@ -119,6 +119,7 @@ DpdkDevice::DpdkDevice(int port, uint32_t mBufPoolSize, uint16_t mBufDataSize)
 	std::ostringstream deviceNameStream;
 	deviceNameStream << "DPDK_" << m_Id;
 	m_DeviceName = deviceNameStream.str();
+	m_DeviceSocketId =  rte_eth_dev_socket_id(m_Id);
```

Comment:
nit: you have an extra space 🙂

Reply:
@seladb  I just fixed it, so should be okay to merge.